R/cal-estimate.R

```diff
@@ -0,0 +1,405 @@
+#------------------------------ Estimates --------------------------------------
+#----------------------------- >> Logistic --------------------------------------
+#' Uses a logistic model to calibrate probabilities
+#' @param .data A data.frame object containing predictions and probability columns.
+#' @param truth The column identifier for the true class results
+#' (that is a factor). This should be an unquoted column name.
+#' @param estimate A vector of column identifiers, or one of `dplyr` selector
+#' functions to choose which variables contains the class probabilities. It
+#' defaults to the prefix used by tidymodels (`.pred_`). The order of the
+#' identifiers will be considered the same as the order of the levels of the
+#' `truth` variable.
+#' @param ... Additional arguments passed to the models or routines used to
+#' calculate the new probabilities.
+#' @examples
+#' # It will automatically identify the probability columns
+#' # if passed a model fitted with tidymodels
+#' cal_estimate_logistic(segment_logistic, Class)
+#' # Specify the variable names in a vector of unquoted names
+#' cal_estimate_logistic(segment_logistic, Class, c(.pred_poor, .pred_good))
+#' # dplyr selector functions are also supported
+#' cal_estimate_logistic(segment_logistic, Class, dplyr::starts_with(".pred_"))
+#' @export
+cal_estimate_logistic <- function(.data,
```

Comment:
I think that we should merge `cal_estimate_logistic()` and the smoothed version (and default for smoothness). It seems like a lot of redundancy.